test/unittests/parser_test.cpp

```diff
@@ -9,6 +9,15 @@ static const auto functype_void_to_void = from_hex("600000");
 static const auto functype_i32i64_to_i32 = from_hex("60027f7e017f");
 static const auto functype_i32_to_void = from_hex("60017f00");
 
+namespace
+{
+bytes make_section(uint8_t id, const bytes& content)
+{
+    assert(content.size() < 0x80);
+    return bytes{id} + uint8_t(content.size()) + content;
```

Comment:
```suggestion
    return bytes{id, uint8_t(content.size()} + content;
```

@chfast do you want to merge it like this?

Reply:
Yes, this is fine.
I would also use `static_cast<uint8_t>` explicitly here.